snap/local/mux_select_nav_vel.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+
+$SNAP/opt/ros/underlay_ws/opt/ros/$ROS_DISTRO/bin/ros2 service call /mux/select topic_tools_interfaces/srv/MuxSelect "{topic: /nav_vel}"
```

Comment:
```suggestion
ros2 service call /mux/select topic_tools_interfaces/srv/MuxSelect "{topic: /nav_vel}"
```
As far as I am concerned this is no longer required